config/release_notes.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
Let's make sure we ignore config/ in our gemspec (see how it works for specs). Don't want this packaged with the gem